TELEMETRY.md

```diff
@@ -1913,22 +1913,6 @@ Expand each section to see more information about that event.
         Common measurement used across a number of events.  
 
 
-* DS_INTERNAL.INTERPRETER_LISTING_PERF  (Telemetry.InterpreterListingPerf)  
```

Comment:
This telemetry is no longer valid, as we no longer have an API to fetch all python enviornments.